src/Server.php

```diff
@@ -18,14 +18,26 @@ public function __construct(LoopInterface $loop)
 
     public function listen($port, $host = '127.0.0.1')
     {
-        if (strpos($host, ':') !== false) {
+        $localSocket = '';
+        if (filter_var($host, FILTER_VALIDATE_IP, FILTER_FLAG_IPV4)) {
```

Comment:
@staabm  
This ext. is since PHP 5.2 enabled by default. http://php.net/manual/en/filter.installation.php
I see many methods from multiple PHP ext.s used in react and no entry in composer.json for them.
What is the difference by ext-filter?